ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -316,6 +316,7 @@ enum CocoaPodUtils {
 
     // Include the minimum iOS version.
     podfile += """
+    use_frameworks!
```

Comment:
`use_frameworks!` was purposely not there to build static library frameworks.

Reply:
It seems there's a way for xcframework containers to take either, but I didn't get it to work, since that container then expects just a static binar and not a fake framework with header constructs. With dyld3 being there on iOS 13 that shouldn't be a huge impact, and we can delete quite a bit of the script logic. 

Folks with CocoaPods can still choose either.